tests/test_pattern.py

```diff
@@ -449,9 +449,8 @@ def test_standardize_domains_and_clifford(self, fx_bg: PCG64, jumps: int) -> Non
         pattern.add(C(node=0, clifford=clifford.TABLE[c]))
         pattern_ref = pattern.copy()
         pattern.standardize(method="direct")
-        state_ref = pattern_ref.simulate_pattern()
-        state_p = pattern.simulate_pattern()
-        assert np.abs(np.dot(state_p.flatten().conjugate(), state_ref.flatten())) == pytest.approx(1)
+        _state_ref = pattern_ref.simulate_pattern()
+        _state_p = pattern.simulate_pattern()
```

Comment:
Same as above.

Reply:
Fixed in b79406d: the removal of the assertion was a mistake, indeed.